packages/arch/packages/modal-utils/src/transitions.js

```diff
@@ -197,3 +197,39 @@ export const ZoomInDown = (props: TransitionProps) => {
     />
   );
 };
+
+// Spring Down
+// ------------------------------
+
+export const SpringDown = (props: TransitionProps) => {
+  const base = {
+    transformOrigin: 'top',
+    transitionProperty: 'opacity, transform',
+    transitionDuration,
+    transitionTimingFunction: 'cubic-bezier(0.2, 0, 0.16, 1.6)',
+  };
+  return (
+    <TransitionReducer
```

Comment:
Definitely not all something that we should do in this PR or necessarily soon but I'd love to remove this `TransitionReducer` component at some point and have the api for a transition be `transitionState => styles`

Reply:
God yes. TBH, most of Arch needs an overhaul.